AI/CMakeLists.txt

```diff
@@ -36,6 +36,11 @@ endif()
 #        Add subdirectories           #
 #######################################
 
+if(BUILD_SINGLE_APP)
+	set(VCMI_LIB_AI vcmi_lib_client)
```

Comment:
why it's called LIB_AI if it's not AI-related lib?

Reply:
good point, will double-check the naming